server/src/entity/SoloMarsEventCard.ts

```diff
@@ -0,0 +1,18 @@
+import { Column, Entity, PrimaryGeneratedColumn, ManyToMany, JoinColumn} from "typeorm";
+import { SoloMarsEventDeck } from "@port-of-mars/server/entity/SoloMarsEventDeck";
+
+@Entity()
+export class SoloMarsEventCard {
+  @PrimaryGeneratedColumn()
+  id!: number;
+
+  @Column()
+  description!: string;
+
+  @Column()
+  displayName!: string;
+  
+  @Column()
+  codeName!: string;
+
```

Comment:
We should be able to encode most of the card effects here by storing the ranges of the effect it has on system health, points, etc. Then we'd just need a special case for the 'draw 2' card and potentially the forced investment card (depending on implementation)

Reply:
I added this and committed the changes. Temporarily I set draw2 as a string, since I was not sure what type of special case we wanted to create.

Reply:
Thanks, we'll just check something like `codeName === "murphys-law"`. I'm working on getting the server code up so I'll merge these changes into my branch and try to get enough of the plumbing in place so that the UI can be worked on